src/Domain/CoordinateMaps/Frustum.cpp

```diff
@@ -102,19 +103,19 @@ Frustum::Frustum(const std::array<std::array<double, 2>, 4>& face_vertices,
   w_plus_ = projective_scale_factor + 1.0;
   w_minus_ = projective_scale_factor - 1.0;
   phi_ = transition_phi;
+  half_opening_angle_ = 0.5 * opening_angle;
+  one_over_tan_half_opening_angle_ = 1.0 / tan(half_opening_angle_);
   // The radius is taken to be the distance from the origin to the vertex of
-  // the Frustum that is furthest away from the origin. This vertex is
-  // assumed to lie on a cube that is centered on the origin. Then the
-  // distance from the origin of the vertex is given by the distance along one
-  // of the x, y, or z axes to the vertex, multiplied by sqrt(3), as the
-  // distance from the origin to the vertex along the other two axes are the
-  // same as the first, assuming it is a vertex of the origin-centered cube.
-  radius_ = std::max({abs(upper_x_upper_base), abs(upper_y_upper_base),
-                      abs(upper_x_lower_base), abs(upper_y_lower_base),
-                      abs(lower_x_upper_base), abs(lower_y_upper_base),
-                      abs(lower_x_lower_base), abs(lower_y_lower_base),
-                      abs(upper_bound), abs(lower_bound)}) *
-            sqrt(3.0);
+  // the Frustum that is furthest away from the origin. For the rectangular
+  // BinaryCompactObject Domain, this vertex is assumed to lie on a rectangular
```

Comment:
This is a good question - there is only one other Domain that uses Frustums, the FrustalCloak Domain, but this Domain has not been updated or used in a while. I expect the BinaryCompactObject Domain to be the only domain the typical user uses that would lead them to encounter a Frustum map, which is why I added this comment.

I could add, `For information on how other Domains use the Frustum map, please see that Domain's documentation.` or something to that effect.

Reply:
Yeah I think that comment would be helpful.